src/errors.test.ts

```diff
@@ -1,21 +1,27 @@
-import { SinonSandbox, SinonSpy, SinonStub, assert, createSandbox } from "sinon";
+import * as assert from "assert";
+import * as sinon from "sinon";
```

Comment:
Unfortunately can't use the sinon Assertion API to handle non-stub/-spy `ok`/`equals`/etc